plugins/frontend/backstage-plugin-shortcut/src/components/ComponentExtensions/EntityStoriesCard.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2024 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { ErrorPanel, Table, TableColumn } from '@backstage/core-components';
+import SyncIcon from '@material-ui/icons/Sync';
+import { shortcutApiRef } from '../../api';
+import { useApi } from '@backstage/core-plugin-api';
+import { useAsyncRetry } from 'react-use';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { SHORTCUT_QUERY_ANNOTATION } from '../../constants';
+import { Story, User } from '../../api/types';
+
+const columnsBuilder: (users?: User[]) => TableColumn<Story>[] = (
```

Comment:
I think someone who is better at react could probably make this card look nicer. But I would argue that the function of this card is useful whether ugly or not.